packages/client/hmi-client/src/components/workflow/tera-simulate-chart.vue

```diff
@@ -23,20 +23,72 @@
 
 <script setup lang="ts">
 import _ from 'lodash';
-import { ref, watch, computed, onMounted } from 'vue';
+import { ref, computed, watch, onMounted } from 'vue';
 import MultiSelect from 'primevue/multiselect';
 import Chart from 'primevue/chart';
 import { ChartConfig, RunResults } from '@/types/SimulateConfig';
 
 const emit = defineEmits(['configuration-change']);
 
+const props = defineProps<{
+	runResults: RunResults;
+	chartConfig: ChartConfig;
+	hasMeanLine?: boolean;
+	colorByRun?: boolean;
+}>();
+
 type DatasetType = {
 	data: number[];
 	label: string;
 	fill: boolean;
 	tension: number;
 };
 
+const renderedRuns = computed<RunResults>(() => {
+	if (!props.hasMeanLine) return props.runResults;
+
+	const runResult: RunResults = JSON.parse(JSON.stringify(props.runResults));
```

Comment:
You could use `_.cloneDeep()` has lodash is already imported on the page.

Reply:
Very fair point will change that